### PR TITLE
Update Turbo Button Tutorial: Add other emulators + RetroArch config saving

### PR DESCRIPTION
**Pull request by tomojira**
Added information on BizHawk, DuckStation, PCSX2, and PPSSPP. At the suggestion of Nepiki, added a small section explaining how to save RetroArch's configuration as well. Also I realized there that the Turbo Fire menu in RetroArch is found in 2 places, so I added that.

The title still singles out RetroArch, but that may be fine if that's mainly what people are searching for, and it's the majority of the tutorial anyway. The other emulators are relatively straightforward.

### CODE REVIEW

File: docs/general/tutorials/turbo-button.md [hunk 1]
```diff
@@ -105,3 +125,45 @@ Under Quick Menu -> Core Options -> Input, there is an option to change the turb
 - IV (default: Y)
 
 Under Quick Menu -> Core Options -> Input, the Turbo Hotkey Mode option can enable the III and IV buttons to have turbo functionality. It can be set to Toggle (press III or IV once to cause the II (default: A) or I (default: B) buttons to be turbo when held) or Dedicated (hold III or IV to directly cause a turbo II or I). There is also an Alternate Turbo Hotkey that, when Toggle is set, allows the L3 (Left Thumb) and R3 (Right Thumb) inputs to toggle the turbo functionality for II and I. The Turbo Delay option can change the frequency of the turbo inputs.
+
+## BizHawk
+
+BizHawk uses the term autofire for this feature - turbo instead refers to a fast forward. While a game is running, you can access autofire configuration under this menu:
```
Comment (wescopeland) [inline docs/general/tutorials/turbo-button.md]:
```suggestion
BizHawk uses the term autofire for this feature - turbo instead refers to increasing the emulation speed. While a game is running, you can access autofire configuration under this menu:

```

File: docs/general/tutorials/turbo-button.md [hunk 2]
```diff
@@ -105,3 +125,45 @@ Under Quick Menu -> Core Options -> Input, there is an option to change the turb
...
+
+## BizHawk
+
+BizHawk uses the term autofire for this feature - turbo instead refers to a fast forward. While a game is running, you can access autofire configuration under this menu:
+
+1. Config
+2. Controllers
+3. Autofire Controls tab
+
+Autofire can be assigned for any button in the current core.
```
Comment (wescopeland) [inline docs/general/tutorials/turbo-button.md]:
```suggestion
Autofire can be assigned to any button in the current core.

```

**Review by wescopeland — COMMENTED** [review]
Thank you! Just a few minor nits here. The only other thing I recommend is, if possible, add screenshots for the other emulators kinda like I did for RetroArch.

If you need any help with this, I'd be happy to assist. Just let me know.

**Comment by tomojira** [comment]
Sure, I'll add some screenshots.

**Review by wescopeland — APPROVED** [review]

**Pull request merged**